presto-main/src/test/java/com/facebook/presto/block/TestArrayBlock.java

```diff
@@ -110,6 +112,58 @@ public void testWithArrayBlock()
         assertBlockFilteredPositions(expectedValuesWithNull, blockBuilderWithNull.build(), () -> blockBuilder.newBlockBuilderLike(null), 2, 3, 4, 9, 13, 14);
     }
 
+    @Test
+    public void testSingleValueBlock()
+    {
+        // 1 entry array.
+        long[][] values = createTestArray(50);
+        // ArrayBlock's hold onto the offsets from builder. MapBlock copies the required portion of the offsets into a new array.
```

Comment:
1. Rewrap this comment into two lines. 
2. `ArrayBlock's hold` looks like a possessive voice, see if it would look better to you if change it to 'ArrayBlock holds` similar to MapBlock in the next sentence.